packages/jmix-front-generator/src/generators/react-typescript/app/template/i18n-message-packs/zh_cn.json

```diff
@@ -14,6 +14,7 @@
 
   "common.alt.logo": "Logo",
   "common.ok": "确认",
+  "common.clone": "克隆",
```

Comment:
We don't add messages in foreign languages, if not sure about the translation. Better use `"common.clone [TRANSLATION NOT AVAILABLE]"` as value. The same in other languages.